### ISSUE TITLE
Converting a time zone should be done by changing region, not with UTC and time zone funcs

### ISSUE THREAD

**Issue by Hout** [malcommac/SwiftDate#99]
Currently the funcs `fromTimeZone` and `inUTCDate` complicate conversions. Converting should be as easy as:
```
let date1 = NSDate()

// I want to know the time here locally and in China
let china = Region(timeZone: chinaTimeZone)
let date2 = date1.inRegion(china)

// Now let us see what we have
print(date1.hour)  // prints the local hour
print(date2.hour)  // prints the hour in china in our calendar and locale
```
Mind that this is pseudo code to illustrate the issue.

**Comment by malcommac** [comment]
So should be like current inRegion() function but it takes directly the timezone to create a default region without passing other optional params?

**Comment by Hout** [comment]
Correct, as implemented in fd73ef7bb9b28d69859260407acb6c01858b48f2

**Comment by Hout** [comment]
Fixed in v3.0.0
